WordPress/src/main/java/org/wordpress/android/ui/stats/refresh/lists/sections/insights/usecases/PostingActivityMapper.kt

```diff
@@ -29,33 +29,38 @@ class PostingActivityMapper
         for (month in months) {
             val firstDayOfMonth = Calendar.getInstance(localeManagerWrapper.getLocale())
             firstDayOfMonth.set(month.year, month.month, firstDayOfMonth.getActualMinimum(Calendar.DAY_OF_MONTH))
+            val getMonthDisplayName = { style: Int ->
+                firstDayOfMonth.getDisplayName(
+                        Calendar.MONTH,
+                        style,
+                        localeManagerWrapper.getLocale()
+                )
+            }
             val firstDayOfWeek = Calendar.getInstance(localeManagerWrapper.getLocale())
             firstDayOfWeek.time = firstDayOfMonth.time
             firstDayOfWeek.set(Calendar.DAY_OF_WEEK, firstDayOfWeek.firstDayOfWeek)
             val boxes = mutableListOf<Box>()
             while (firstDayOfWeek.before(firstDayOfMonth)) {
-                boxes.add(INVISIBLE)
+                boxes.add(Box(INVISIBLE))
                 firstDayOfWeek.add(Calendar.DAY_OF_MONTH, 1)
             }
             for (day in month.days) {
                 boxes.add(
-                        when {
-                            day.value > veryHighLimit -> VERY_HIGH
-                            day.value > highLimit -> HIGH
-                            day.value > mediumLimit -> MEDIUM
-                            day.value > LOW_LEVEL -> LOW
-                            else -> VERY_LOW
-                        }
+                        Box(
+                                when {
+                                    day.value > veryHighLimit -> VERY_HIGH
+                                    day.value > highLimit -> HIGH
+                                    day.value > mediumLimit -> MEDIUM
+                                    day.value > LOW_LEVEL -> LOW
+                                    else -> VERY_LOW
+                                }, day.key
+                        )
                 )
             }
             blocks.add(
                     Block(
-                            firstDayOfMonth.getDisplayName(
-                                    Calendar.MONTH,
-                                    Calendar.SHORT,
-                                    localeManagerWrapper.getLocale()
-                            ),
-                            boxes
+                            getMonthDisplayName(Calendar.SHORT),
```

Comment:
We might want to consider settings the content description to something like "Posting Activity for $getMonthDisplayName". Wdyt?

Reply:
That sounds good! I will look into it.

Reply:
Resolved 40cd081